e2e/paths/enterFactorsAffectingParenting_test.js

```diff
@@ -0,0 +1,26 @@
+const config = require('../config.js');
+
+Feature('EnterFactorsAffectingParenting');
+
+Before((I, caseViewPage) => {
+	I.logInAndCreateCase(config.localAuthorityEmail, config.localAuthorityPassword, config.eventSummary, config.eventDescription);
+	I.waitForElement('.tabs', 10);
+	caseViewPage.goToNewActions(config.applicationActions.enterParentingFactors);
+	I.waitForElement('ccd-case-edit-page', 5);
+});
+
+Scenario('Leaving factors affecting parenting section blank in c110a', (I) => {
+	I.continueAndSubmit(config.eventSummary, config.eventDescription);
+	I.see(`updated with event: ${config.applicationActions.enterParentingFactors}`);
+});
+
+Scenario('There is an optional textArea when I answer yes to a question in factors affecting parenting section', (I, enterFactorsAffectingParentingPage) => {
```

Comment:
It does not sound like test scenario, it is worth renaming.

Reply:
before I change, 'Clicking yes to a question produces a textarea in factors affecting parenting section'. Does this sound better to you?